src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -899,6 +901,9 @@ int MPIDI_OFI_flush_send_queue(void)
   fn_exit:
     return mpi_errno;
   fn_fail:
+    if (in_vci_cs != -1) {
```

Comment:
Do we need to reset to `-1` after the loop? Otherwise won't it double unlock the last vci in the loop?

Reply:
The last loop reaches the `fn_exit` label.